types/src/front/lib/assistant.ts

```diff
@@ -244,6 +248,36 @@ export const GPT_4O_MINI_MODEL_CONFIG: ModelConfigurationType = {
   toolUseMetaPrompt: LEGACY_OPEN_AI_TOOL_USE_META_PROMPT,
   supportsVision: true,
 };
+export const O1_PREVIEW_MODEL_CONFIG: ModelConfigurationType = {
+  providerId: "openai",
+  modelId: O1_PREVIEW_MODEL_ID,
+  displayName: "Open AI O1 Preview",
```

Comment:
No space in OpenAI :)

Reply:
Also we generally don't put it in the displayName since the logo is next to it?

Reply:
https://github.com/dust-tt/dust/pull/7512